### PR TITLE
docs: document query call members method

### PR DESCRIPTION
**Pull request by szuperaz**
https://getstream.slack.com/archives/C06ALLF4QNB/p1727120067644979

### CODE REVIEW

File: packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+id: querying-call-members
+title: Querying Call Members
+description: How to query call members
+---
+
+import FilterConditions from '../../../shared/_filter-operators.mdx';
+import CallMemberFilters from '../../../shared/video/_call-member-filters.mdx';
+import CallMemberSort from '../../../shared/video/_call-member-sort-fields.mdx';
```
Comment (kristian-mkd) [inline packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx:9]: Should these be wrapped in ```typescript ...``` wrapper?

Comment (szuperaz) [reply packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx:9]: No, they work like this, I checked by locally running the docs site

Comment (kristian-mkd) [reply packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx:9]: Nice, I was checking this with the `preview file` option here.

**Review by kristian-mkd — APPROVED** [review]

File: packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```typescript
+await call.queryMembers();
+
+// sorting and pagination
+const queryMembersReq = {
+  sort: [{ field: 'user_id', direction: 1 }],
+  limit: 2,
+};
+await call.queryMembers(queryMembersReq);
```
Comment (myandrienko) [inline packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx:25]: Here and in similar places: `const result =`?

**Review by myandrienko — APPROVED** [review]

**Pull request merged**